text/chapter6.md

```diff
@@ -659,8 +659,6 @@ Another reason to define a superclass relationship is in the case where there is
      ```
      _Hint_: Search Pursuit for a helper-function with the signature [`String -> Int -> String`](https://pursuit.purescript.org/search?q=String%20-%3E%20Int%20-%3E%20String). Note that `String` might appear as a more generic type (such as `Monoid`).
 
-     Does this instance satisfy the laws listed above?
```

Comment:
I believe the goal here is to call attention to abiding to typeclass laws. This is one of the _weak_ parts of the language were the type system and the compiler does not help. It's merely a hand-shake contract between authors. So I'm against removing this reminder altogether.

Some alternatives
  - "Convince yourself that this instance satisfies the laws listed above."
  - "Note that this instance satisfies the laws listed above."

However, I like the original _question_ form in that it naturally stimulates curiosity. It becomes a personal mission to find out.

Reply:
In that case, then why not also include this reminder a third time in the next exercise, and maybe a fourth time in the exercise after that too?

My argument against including the reminder here is that it seems redundant after the reminder in the first exercise.

Reply:
I think twice is quite enough. For a reader, the first time brings about the idea. The second time hardens it. The third time, well, starts to seem redundant. The fourth time, it's either, this is getting annoying, or, starting to think this must be of supreme importance! hence the repetition.

The theme of this book is "by example". Not everything is explained to the detail. Prior to these exercises, typeclass laws were mentioned only briefly (first time when introducing Functor laws, and second time when mentioning super class relation). Then jumps into implementing a typeclass with laws. So for the first time, definitely stress that laws must be satisfied. The second time, a gentler reminder. Twice should be enough to set the pattern for the subsequent times. If not, the tests will kick in.

So I stand by the use of phrases "Remember, ..." for the first time, and "Does this instance satisfy..." for the second time. And not for subsequent exercises.